apollo-ios-pagination/Sources/ApolloPagination/AsyncGraphQLQueryPager.swift

```diff
@@ -237,6 +237,8 @@ public class AsyncGraphQLQueryPager<Model>: Publisher {
   /// Resets pagination state and cancels further updates from the pager.
   public func cancel() async {
     await pager.cancel()
+    _subject.send(completion: .finished)
```

Comment:
Do we need to update unit tests for this? Can also be done in a separate PR later.

Reply:
I think that'd be wise, yes. 
As written, I think this would require us to re-init a new pager – in the same sense that cancelling a `watcher` stops it from being used again.